app/src/shared/utils/request.js

```diff
@@ -17,8 +18,11 @@ export const getError = (res: any): ErrorInUi => ({
 })
 
 export default function request(url: string, method: RequestMethod = 'get', data?: any = null, options?: Object): ApiResult<*> {
+    const token: ?string = SessionProvider.token()
     const defaultOptions = {
-        headers: {},
+        headers: {
+            Authorization: `Bearer ${token || 0}`,
```

Comment:
@juhah, this is a hack that forces the backend to think that we're authorizing via the token, not cookie-based sessions.